src/containers/WebComponentLoader.test.js

```diff
@@ -147,7 +155,7 @@ describe("When loaded as a cold user", () => {
 
     test("Calls useProjectPersistence hook with correct attributes", () => {
       expect(useProjectPersistence).toHaveBeenCalledWith({
-        user,
+        user: undefined,
         project: { components: [] },
         hasShownSavePrompt: false,
         justLoaded: false,
```

Comment:
I'm a bit confused by this bit - why are we expecting the user to be undefined even though they're logged in? Is it because the user only gets set when the user attempts to call an action, but not initially? 🤔

Reply:
Yea its exactly that. So all we can really test is calls are made to the dispatch functions with the inputs we expect, they don't actually change state. So this will always be `undefined` (notice it's not `null` which it would be off that initial `setUser` action). So I've added the tests to check for the `setUser` dispatches here too